doc/api/perf_hooks.md

```diff
@@ -98,6 +98,43 @@ added: v8.5.0
 Returns the current high resolution millisecond timestamp, where 0 represents
 the start of the current `node` process.
 
+### performance.startEventLoopTimer([options])
```

Comment:
This is not part of the W3C Performance Timeline spec as what is claimed at the top of the documentation...can we document the divergence in a more significant manner?

Reply:
Likely a good idea, but would prefer to do so in a separate PR

Reply:
Moved the functions to the `perf_hooks` top level rather than hanging them off `Performance`